src/main/java/forceitembattle/commands/CommandInfo.java

```diff
@@ -28,6 +29,12 @@ public boolean onCommand(CommandSender commandSender, Command command, String s,
             return false;
         }
 
+        ForceItemBattle.getItemDifficultiesManager().getDescriptionItems().forEach(items -> {
+            if(items.material() == item.getType()) {
+                ForceItemBattle.getItemDifficultiesManager().getDescriptionItem(items.material()).forEach(player::sendMessage);
```

Comment:
same as before... could this cause issues if the item description is "null"? Maybe just check and send an error in case.

Reply:
Yes thats true, good point